### PR TITLE
Session02 - FizzBuzz Exercise

### PR DESCRIPTION
**Pull request by MadhuAcharya123**
Session-02 Exercises.

### CODE REVIEW

**Comment by MadhuAcharya123** [comment]
Session - 02 Exercises

**Comment by MadhuAcharya123** [comment]
Session - 02 Exercises

File: students/madhu/FizzBuzz.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+   for i in range(1,101):
+       mod3 = i % 3
+       mod5 = i % 5
+       mod15 = i % 15
+
+       if mod3 == 0 and mod15 != 0:
+           print ("Fizz")
+       elif mod5 == 0 and mod15 != 0:
+           print ("Buzz")
+       elif mod15 == 0:
```
Comment (PythonCHB) [inline students/madhu/FizzBuzz.py]: if you check mod15 first, then the logic can be less complex.

File: students/madhu/session02/GridPrinterExercise.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+def print_grid2(rows,cell_size):
+
+    plus = '+'
+    minus = '- '
+    length = cell_size + (cell_size - 1) + 2
+    bars = '|' + (' ' * length)
+    ends = '+ ' + ('- ' * cell_size)
+
+    for i in range(0,rows):
```
Comment (PythonCHB) [inline students/madhu/session02/GridPrinterExercise.py]: you can just do ``range(rows)`` -- the zero is assumed.

File: students/madhu/session02/series.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
+def fibonacci(n): #function to print out the nth element in the fibonacci series#
+    """Return the nth element in the fibonacci series."""
+    lst = []
+    lst.insert(0,0)
```
Comment (PythonCHB) [inline students/madhu/session02/series.py]: if you are tacking stuff on the end -- easier to use ``list.append()``

File: students/madhu/session02/series.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    for i in range(2,n+1):
+        value = lst[i-1] + lst[i-2]
+        lst.insert(i,value)
+
+    print ("The nth number in the series is: " + str(lst[n]))
+
+
+
+if __name__ == "__main__":
+    fibonacci(10) #calling the fibonacci function to return the 10th element in the series#
```
Comment (PythonCHB) [inline students/madhu/session02/series.py]: ideally, you can call the functions and check that the results are correct:

``assert fibonacci(10) == 55``

**Pull request merged**